.kokoro/release/stage.sh

```diff
@@ -27,11 +27,14 @@ pushd $(dirname "$0")/../../
 setup_environment_secrets
 create_settings_xml_file "settings.xml"
 
-echo "Deploying artifacts to staging repositories"
```

Comment:
No functionality change in this shell script. Just changing variable name and comments.